src/DomCrawler/Crawler.php

```diff
@@ -34,7 +34,7 @@ final class Crawler extends BaseCrawler implements WebDriverElement
     /**
      * @param WebDriverElement[] $elements
      */
-    public function __construct(array $elements = [], WebDriver $webDriver, ?string $uri = null)
+    public function __construct(array $elements = null, WebDriver $webDriver, ?string $uri = null)
```

Comment:
This doesn't look good to me. It Should be:
```suggestion
    public function __construct(array $elements = [], ?WebDriver $webDriver = null, ?string $uri = null)
```

Reply:
See #393